packages/orchestration/README.md

```diff
@@ -125,6 +125,28 @@ console.log(
 );
 ```
 
+> [!Tip]
+>
+> #### Harmonized API
+>
+> You can switch to a different model, even from another vendor, by leveraging the harmonized API of the orchestration service.
+> Here’s an example where only one line of code is changed..
```

Comment:
```suggestion
> Here’s an example where only one line of code is changed.
```

Reply:
[pp] Not sure if it is necessary to provide a complete example here just for changing a property. It feels hard to emphasize this advantage on written text unlike seeing a line change in a real demo.